package.json

```diff
@@ -114,6 +114,15 @@
 					"title": "Custom Fonts",
 					"description": "Configures the Draw.io editor to use custom fonts."
 				},
+				"hediet.vscode-drawio.customPlugins": {
+					"type": "array",
+					"items": {
+						"type": "string"
+					},
+					"default": [],
+					"title": "Custom Plugins",
+					"description": "Loads custom scripts from specified URIs to extend the Draw.io editor"
```

Comment:
Thanks for specifying. But I think URIs have to be absolute. How does that work?

Reply:
by specifying an absolute uri? what do you mean?

Reply:
I think it would be great if local plugins of a workspace could be loaded (these settings can be configured on a per workspace level).
So I think you don't really need to parse the value as URIs  or even accept URIs here. Local paths that are resolved to the workspace root of the diagram (that may lead to no file-URIs) and absolute file paths (that always lead to file-URIs) should be enough here. That should align with how custom libraries are loaded.

Reply:
This is how I do it for custom libraries:

https://github.com/hediet/vscode-drawio/blob/01b5f9b94f25a1fb6aae50ac2e0b31631a42c134/src/Config.ts#L178

Reply:
Initially they were absolute paths. I switched over to uris to test if external resources (http) would be loadable. As far as I see, this is not the case?

Reply:
I guess not and I think it should be implemented similar to how I already implemented custom libraries:
support `{ file: "C:\\test.js" }`, `{ file: "${workspaceFolder}\\test.js" }`, `{ file: "${workspaceFolder}/test.js" }`, or `{ url: "https://my-site/plugin.js" }` or even `{ builtin: "foo" }`

Reply:
When only local resources are allowed why go through this trouble?
A simple check if the path is absolute should suffice, shouldn't it?

Reply:
I tried to align with how `launch.json` in VS Code works. See their `program` property (https://code.visualstudio.com/docs/editor/debugging).
They do it like this:
```
"program": "${workspaceFolder}/node_modules/gulp/bin/gulpfile.js",
```

I guess just not allowing relative paths, but providing `${workspaceFolder}` makes it harder to mess things up.

Reply:
how does this resolving happen? can't find any api that provides this.

Reply:
There is no api, use the function I added.